setup.cfg

```diff
@@ -1,10 +1,66 @@
 [metadata]
-package_name = jwst
-description = JWST Pipeline
-author = STScI
-license = BSD
-edit_on_github = False
-github_project = spacetelescope/jwst
+name = jwst
+description = Python library for science observations from the James Webb Space Telescope
+long_description = file: README.md
+long_description_content_type = text/markdown
+author = JWST Pipeline developers
+license = BSD 3-Clause License
+url = https://github.com/spacetelescope/jwst
+classifiers =
+    Intended Audience :: Science/Research
+    Topic :: Scientific/Engineering :: Astronomy
+    License :: OSI Approved :: BSD License
+    Programming Language :: Python :: 3
+
+[options]
+zip_safe = False
+python_requires = >=3.6
+setup_requires =
+    setuptools_scm
+install_requires =
+    asdf>=2.5
+    astropy>=4.0
```

Comment:
Perhpas set this to `4.0.1.post1`. There was an issue with 4.0.

Reply:
It was some install issue for certain platforms, right?  We currently test with 4.0.0 on Linux without problem, but perhaps other platforms (Windows) it is not good?

Reply:
I would have to check, but I think James White added some type notation. Is python 3.6 still compatible with that?